2216/2/battle.rb

```diff
@@ -0,0 +1,34 @@
+class Battle
```

Comment:
Battle class should represent one battle.
See my earlier comment: https://github.com/rubizza-survival-camp/summer-2018/pull/219#issuecomment-406068631